app/assets/stylesheets/pages/_partners.scss

```diff
@@ -0,0 +1,30 @@
+a:hover {}
+
+ a,
```

Comment:
this is changing every single "a" element in our app. maybe previous elements already have a different style that we do not want to change. give a different class and never target a default element